tests/st/calicoctl/test_crud.py

```diff
@@ -1479,369 +1479,366 @@ def test_get(self):
+                                         'metadata': {},
+                                         'spec': {'ipip': {'mode': 'Always'},
+                                                  'cidr': "10.0.1.0/33"}  # impossible mask
+                                         }),
+                   ("pool-invalidNet2", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
+                                         'spec': {'ipip': {'mode': 'Always'},
+                                                  'cidr': "10.0.256.0/24"}  # invalid octet
+                                         }),
+                   ("pool-invalidNet3", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
+                                         'spec': {'ipip': {'mode': 'Always'},
+                                                  'cidr': "10.0.250.0"}  # no mask
+                                         }),
+                   ("pool-invalidNet4", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
+                                         'spec': {'ipip': {'mode': 'Always'},
+                                                  'cidr': "fd5f::2::1/32"}  # too many ::
+                                         }),
+                   #  https://github.com/projectcalico/libcalico-go/issues/224
+                   # ("pool-invalidNet5a", {'apiVersion': API_VERSION,
+                   #                       'kind': 'IPPool',
+                   #                       'metadata': {'cidr': "::/0"},  # HUGE pool
+                   #                       }),
+                   # ("pool-invalidNet5b", {'apiVersion': API_VERSION,
+                   #                       'kind': 'IPPool',
+                   #                       'metadata': {'cidr': "1.1.1.1/0"},  # BIG pool
+                   #                       }),
+                   ("pool-invalidNet6", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
+                                         'spec': {'cidr': "::/128"}
+                                         # nothing
+                                         }),
+                   ("pool-invalidNet7", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
+                                         'spec': {'cidr': "192.168.0.0/27"}  # invalid mask
+                                         }),
+                   ("pool-invalidNet8", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
```

Comment:
not having a name in the metadata will likely be the reason for this failing.  Same above.

Reply:
and below.

Reply:
`scope` is also no longer a valid field in the `Spec` right?